x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -147,6 +147,8 @@ export const showAllOthersBucket: string[] = [
 
 /**
  * CreateTemplateTimelineBtn
+ * https://github.com/elastic/kibana/pull/66613
  * Remove the comment here to enable template timeline
  */
-export const disableTemplate = true;
+export const disableTemplate = false;
```

Comment:
I am wondering if we still need the flag. Maybe, we can create another PR later on to remove this flag.

Reply:
yeah, in another PR sounds good.